api/db/changelog/db.changelog-43-add-page-visit.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+    <changeSet author="ursas" id="changelog-43-add-user-visited-page">
+        <createTable tableName="page_visit">
+            <column name="page_visit_id" type="bigint" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="user_id" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+            <column name="page_id" type="varchar(40)">
```

Comment:
Throwing out some random thoughts ... 

What do you think about size here? Here's one url I have on test (length 91):
`https://all-of-us-workbench-test.appspot.com/workspaces/aou-rw-test-139-2/pendingworkspace`

Trimming that down to just the path (length 47):
`/workspaces/aou-rw-test-139-2/pendingworkspace`

Unless this ends up being a join and we normalize the path to another table for `page_id` -> `path`